app/services/UserAnswersService.scala

```diff
@@ -79,7 +80,11 @@ class UserAnswersService @Inject() (
       case Some(psaId) =>
         pensionSchemeConnector.checkPsaAssociation(sessionData.schemeInformation.srnNumber.value, psaId).flatMap {
```

Comment:
We don't need both checks here. isAuthorisingPsa is the correct check to make at this stage.

Reply:
I checked this with Salim and he checked with Ian, there is a different error we show the user if the PSA is not the authorising PSA to if the PSA is not associated with the scheme, it's in the ACs: https://jira.tools.tax.service.gov.uk/browse/OAOTC-1640